### DISCUSSION TITLE
KEEP OLD SECTIONS!!!!!

### DISCUSSION THREAD

**Discussion by jennylens** [elementor/elementor#23806] [Feature Request]
I left a long msg for Elementor developers and programmers ONLY. I don't want nor need comments from their fanboys. So I deleted it. 

Obvious to me no one gives a damn about CONTENT CREATORS who don't have time nor interest using new features. Containers and flexible blocks are not needed IF someone has UNIQUE content that people actually will pay to read about, see or buy prints. 

YOU all try staying on top of Photoshop (2024 beta just released), Lightroom, all of Affinity, Screenflow, Scrivener, and tons more complex software programs that REAL content creators use on a daily basis. THEN add complex Elementor updates. 

Seems like NO consideration of us who want, use and LOVE Elementor BASICS.

IF Elementor insists on making things more complicated, that's great for programmers. But they need to change their marketing. Can't sell to folks who don't have time nor interest to constantly relearn everything cos they are busy teaching, making art, doing things with lots of content. Can't have it both ways. That is all I'm saying!

So goodbye all! Have fun!

**Comment by loucasl** [comment]
No one is forcing you to use containers. Elementor has said multiple times in multiple places that even though containers are the future, sections are not being removed and you can TURN OFF the container feature in your settings.

**Comment by jennylens** [comment]
loucasl: pls tell that to Elementor's twitter person. Took several tweets back and forth before he told me where to turn off those alpha and beta features. AND I've been actively involved in digital tech since 1988. Long enough to know that companies make promises that many later break. 

Although YOU have seen all this info, I have not!!! As I said, I'm not a programmer. When you work with as much data aka photos, art, motion graphics, social media, all the copywriting and NO AI cos I'm a DIY artist, writer and teacher, not on my radar to keep up with every bloody change of every piece of software. Try walking on another's person's shoes before coming down so hard on them.

When you get a BIG NASTY warning on your Plugins to update Elementor and ElementorPro, THAT is the place to tell us we can TURN them off in Settings --> Features. I THINK that's where it is, too busy to launch and look it up. 

When I used Divi, they completely changed the back end three times. That's when a former Divi designer turned me onto Elementor. And that's only one of many companies who DID totally change things. Let's talk about WordPress itself! Gutenberg turned things upside down.

I have rational reasons for not believing companies when they say they won't change things. But IF they hear from PAYING customers, they might take our needs into consideration.

AND YOU DO NOT HAVE TO BE SO NASTY!!!!

**Comment by tobias992** [comment]
@jennylens 

Sorry, but what's wrong with you? You act here like a offended child. And I tell you my point of view:

Exactly you are the customer I don't like. Never move with the times and never want to learn anything new. It's a shame that Elementor puts so much emphasis on backwards compatibility. The whole Elementor plugin could be much better if they focused on modern technologies and not whining after people like you.
